join/free_trial.md

```diff
@@ -5,15 +5,16 @@ Register for openEO platform and immediately start a 30 day free trial:
 
 <a href="https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://openeo.cloud/welcome-to-openeo-platform/&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid" class="action-button" style="display: inline-block; font-size:1.2rem; color: #fff; background-color: #335e6f; padding: 0.8rem 1.6rem; border-radius: 4px; border-bottom: 1px solid #2e5564;">&rarr; Register for openEO Platform</a>
 
+Free trial users receive **1000 free credits** upon registration. openEO platform is a federation of services which include:
+- EGI for Authentification & Authorization via [EGI Check-in](https://www.egi.eu/services/check-in/)
+- Terrascope EOPlaza for Account management
+- EODC for provisioning of JupyterLab
+- SentinelHub, EuroDataCube and Creodiaz backends
```

Comment:
Not sure what you wanted to say here but this is not correct. I would change it to: `Terrascope including Sentinel Hub connection via the EuroDataCube, EODC and Sentinel Hub backends`


- EDC is not an independent backend
- Creodia**S** is not part of the federation (you can find the updated federation graphics [here](https://github.com/openEOPlatform/documentation/blob/main/federation/federation.png))
- you forgot your backend

Reply:
Changed.